src/main/java/org/spongepowered/api/item/inventory/ItemStackComparators.java

```diff
@@ -37,56 +36,69 @@
 public final class ItemStackComparators {
 
     /**
-     * Compares ItemStacks based on {@link ItemType}. This comparator will not
-     * return the same results as ItemStack.equals(ItemStack) for ItemStacks
-     * with extra attached data, different damage values, or different sizes.
+     * Compares ItemStacks based on {@link org.spongepowered.api.item.ItemType}.
```

Comment:
`ItemType` and `ItemStack` are both used more than once.
In addition to that `ItemStack` is also used as return type.

I haven't seen any warnings from gradle nor my IDE here.